### DISCUSSION TITLE
playlist order

### DISCUSSION THREAD

**Discussion by pumo2019** [AzuraCast/AzuraCast#7471] [Q&A]
I have 23 albums/folders which I like to play in order. those are named  album 0 - something, album 1- something else, etc.
under folder files are 01 - title.mp3,  02 -title2.,mp3 etc. id3 tags are done correct way so title is right plays track in right place.
I made playlists each folder but it plays randomly even I enable only one playlist.
in play list option there is ticks general rotation and sequential.
is it possible to play ordered ?

**Comment by Heimatradio** [comment]
The easiest way:
change the track numbers, for example:
Album 01: 01-01-title ... 01-15-title
Album 02: 02-01-title ... 02-12-title
etc.
Then put everything into one folder and playlist, playlist-setting to order

**Reply by pumo2019** [reply]
ok thanks. I have to  think can it renaming scripted, there is quite many playable items :)  . 
I find reordering from gui,  I test can it be used  by  weighting playlist differently so album1: 1 album2:  2  etc.

**Comment by pumo2019** [comment]
I dont understand that logic at all.
What I did: reorder all playlist as should be, but those playlist type advanced and sequential, I added to liquidsoap config page
radio = rotate(id="standard_playlists", weights=[3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3], [playlist_album1, playlist_album2, playlist_album3, all others etc])
now it just plays playlist_album12  all the  time and after restart. I tested playlist priority and it wont affect  at all no its for first  5 1-5 and all others 1
so my next plan is to mp3wrap those albums to one single mp3 file, maybe it works.. sad to loose those tracks but renaming is too big project.
edit: after rebooting whole server not it started with album4
